corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap5/modal_solutions_feature_request.html

```diff
@@ -0,0 +1,119 @@
+        <div class="modal-content">
+          <div class="modal-header">
+            <h1 class="modal-title fs-5" id="requestAFeatureModalLabel">{% trans "Make a Feature Request to Solutions" %}</h1>
+            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+          </div>
+          <div class="modal-body">
+            <p>
+              <strong>{% trans "Please submit this report from the page where you would like to see the change" %}</strong>
+            </p>
+            <div class="form-horizontal">
+              <div class="row mb-3">
+                <label for="request-report-subject"
+                       class="col-sm-2 col-form-label">{% trans "Short Description" %} *</label>
+                <div class="col-sm-10">
+                  <input type="text"
+                         class="form-control"
+                         name="subject"
+                         id="request-report-subject"
+                         placeholder="{% trans 'One sentence description of feature/change request.' %}">
+                  <span class="badge badge-danger d-none">
+                    <i class="fa fa-warning"></i> {% trans 'Please give us some information about the feature/change request' %}
+                  </span>
+                </div>
+              </div>
+              <div class="row mb-3">
+                <label for="request-report-message"
+                       class="col-sm-2 col-form-label">
+                  {% trans "Full Description" %}
+                </label>
+                <div class="col-sm-10">
+                  <textarea name="message"
+                            class="form-control vertical-resize"
+                            id="request-report-message"
+                            rows="3"
+                            placeholder="{% trans "Please describe the product changes you would like to see in CommCare." %}"></textarea>
+                </div>
+              </div>
+              <div class="row mb-3"
+                   id="request-report-cc-form-group">
+                <label for="request-report-cc"
+                       class="col-sm-2 col-form-label">
+                  {% trans "Other recipients" %}
+                </label>
+                <div class="col-sm-10">
+                  <input type="text"
+                         name="cc"
+                         class="form-control"
+                         id="request-report-cc"
+                         placeholder="ex: admin@commcarehq.org, test@commcarehq.org" />
+                  <p class="help-block">
+                      {% blocktrans %}
+                          Comma-separated email addresses of others you want to notify about this request.
+                      {% endblocktrans %}
+                      <span class="badge bade-danger hide">
+                          <i class="fa fa-warning"></i> {% trans 'Incorrect Format' %}
```

Comment:
how is the format validated for emails?

Reply:
I have added the missing JS files. The validation is happening in these files.